Box.V2/Config/BoxConfigBuilder.cs

```diff
@@ -133,6 +134,7 @@ public BoxConfigBuilder SetBoxApiHostUri(Uri boxApiHostUri)
         /// </summary>
         /// <param name="boxAccountApiHostUri">BoxAPI account host uri.</param>
         /// <returns>this BoxConfigBuilder object for chaining</returns>
+        [Obsolete("Use BoxAuthAuthorizeApiUri instead")]
```

Comment:
Shouldn't this Obsolete point to `SetBoxAuthAuthorizeApiUri` method instead of a field?